aptos-move/move-examples/data_structures/sources/bucket_table.move

```diff
@@ -144,7 +144,7 @@ module aptos_std::bucket_table {
     /// Aborts if there is no entry for `key`.
     /// The requirement of &mut BucketTable is to bypass the borrow checker issue described in https://github.com/move-language/move/issues/95
     /// Once Table supports borrow by K, we can remove the &mut
-    public fun borrow<K: copy + drop, V>(map: &mut BucketTable<K, V>, key: K): &V {
+    public fun borrow<K: drop, V>(map: &mut BucketTable<K, V>, key: K): &V {
```

Comment:
why #40 requires copy and here no copy?

Reply:
https://github.com/aptos-labs/aptos-core/pull/6339